frontend/packages/dev-console/integration-tests/features/pipelines/pipelines-runs.feature

```diff
@@ -206,3 +206,16 @@ Scenario Outline: Pipeline status display in topology side bar : P-05-TC02
 Examples:
 | pipeline_name          |
 | pipeline-with-resoruce |
+
+
+@regression, @manual
+Scenario: Tasks unaffected for existing pipeline runs when added a new task in the pipeline
+    Given user has pipeline with two tasks
+    And user is at the Pipeline Details page
+    And pipeline run is present
+    When user clicks Edit Pipeline from Actions menu
+    And user adds a new task in series to the last task present
+    And user clicks on save 
+    And user goes to Pipeline Runs page
+    And user opens the given pipeline run
+    Then user can see two tasks in Pipeline Run details tab
```

Comment:
When user starts the pipeline
Then newly created pipeline run should contain the new pipeline graph
And existing pipeline runs should contain the old pipeline graph

Reply:
Updated